source/Marquee.js

```diff
@@ -1108,4 +1114,26 @@
 		style: 'overflow: hidden;'
 	});
 
+
+	moon.Marquee = {
+		_marqueeOnHoverControl: null,
+
+		setMarqueeOnHoverControl: function(oControl) {
+			this._marqueeOnHoverControl = oControl;
+		},
+
+		getMarqueeOnHoverControl: function() {
+			return this._marqueeOnHoverControl;
+		}
+	};
+
+	enyo.dispatcher.features.push(
+		function (e) {
+			if ("blur" === e.type) {
```

Comment:
You don't test to see where the event came from but blur can happen in other cases so this isn't a safe assumption that any blur would be from the window.